### DISCUSSION TITLE
Does Adnauseam work with Brave's built-in adblocker/shield?

### DISCUSSION THREAD

**Discussion by JcynR** [dhowe/AdNauseam#2253] [Q&A]
Does Adnauseam work with Brave's built-in adblocker and shield, or should this be disabled?
And if it should, which settings should be disabled? What about blocking fingerprinting and trackers?
What is the best settings for Adnauseam + Brave?

Thanks in advance.

**Comment by s38b35M5** [comment]
Bumping for visibility. Assuming like OP that only ad/tracker blocking needs to be disabled to work with Brave, but wondering if the other settings affect AN adversely?

![image](https://github.com/dhowe/AdNauseam/assets/138058715/c0e19912-4fa2-4b89-a85e-d856b61f15ce)

**Reply by s38b35M5** [reply]
Nevermind: https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chromium-based-browsers#optimal-configuration-for-ad-collection-on-brave

**Comment by mneunomne** [comment]
https://github.com/dhowe/AdNauseam/wiki/Install-AdNauseam-on-Chromium-based-browsers#optimal-configuration-for-ad-collection-on-brave answers this
